images/fcos-bastion-image/root/usr/share/bastion_services/nginx/nginx_systemd.conf

```
@@ -0,0 +1,3 @@
+# systemd nginx conf
+PODMAN_NGINX_IMAGE=quay.io/openshifttest/nginx-alpine:armbm
+PODMAN_NGINX_BASEDIR=/var/opt/nginx
```

```suggestion
PODMAN_NGINX_BASEDIR=/var/opt/html
```

To stay in sync with the other code that will add content there.

---

Done